config/sync/media.settings.yml

```diff
@@ -3,4 +3,4 @@ _core:
 icon_base_uri: 'public://media-icons/generic'
 iframe_domain: ''
 oembed_providers_url: 'https://oembed.com/providers.json'
-standalone_url: false
+standalone_url: true
```

Comment:
unset this - it's being set by Isle-DC using code in buildkit. https://github.com/Islandora-Devops/isle-dc/issues/326